client/src/App.js

```diff
@@ -1,116 +1,185 @@
-import { Switch, Route } from 'react-router-dom';
-import React, { useState, useEffect } from 'react';
-import axios from 'axios';
 import styled from 'styled-components';
-import { useLocalStorage } from '../src/hooks/useLocalStorage';
+import { useState, useEffect } from 'react';
+import { Switch, Route } from 'react-router-dom';
 import Home from './Pages/Home';
-import Watchlist from './Pages/Watchlist';
-import Friends from './Pages/Friends';
 import Search from './Pages/Search';
+import axios from './services/axios';
+import Friends from './Pages/Friends';
+import Ratings from './Pages/Ratings';
+import Profile from './Pages/Profile';
+import Watchlist from './Pages/Watchlist';
+import Favorites from './Pages/Favorites';
+import Sidebar from './components/Sidebar';
+import requests from './services/requests';
 import Banner from '../src/components/Banner';
-import Navigation from '../src/components/Navigation';
-import Store from './Store';
 import Searchbar from './components/Searchbar';
+import FriendsCards from './Pages/FriendsCards';
+import ScrollToTop from './services/ScrollToTop';
+import Navigation from '../src/components/Navigation';
+import { useLocalStorage } from '../src/hooks/useLocalStorage';
 
 function App() {
-  const [watchlist, setWatchList] = useLocalStorage('Watchlist', []);
-  const [search, setSearch] = useState([]);
   const [query, setQuery] = useState('');
+  const [open, setOpen] = useState(false);
+  const [search, setSearch] = useState([]);
+  const [show, handleShow] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
-  const baseUrl = 'https://api.themoviedb.org/3/';
-  const APIKEY = process.env.REACT_APP_APIKEY;
+  const [watchlist, setWatchList] = useLocalStorage('Watchlist', []);
+  const [favorites, setFavorites] = useLocalStorage('Favorites', []);
+
+  let fetchUrl;
+  query === ''
+    ? (fetchUrl = requests.fetchTrending)
+    : (fetchUrl = `${requests.fetchSearch}${query}&page=1&include_adult=false`);
```

Comment:
I think you can declare the variable and combine it with the ternary

```suggestion
  const fetchUrl =  query === ''
    ? requests.fetchTrending
    : `${requests.fetchSearch}${query}&page=1&include_adult=false`;
```